app/test/std_sdk_test.go

```diff
@@ -163,7 +163,16 @@ func (s *StandardSDKIntegrationTestSuite) TestStandardSDK() {
 			name: "create legacy community spend governance proposal",
 			msgFunc: func() (msgs []sdk.Msg, signer string) {
 				account := s.unusedAccount()
-				coins := sdk.NewCoins(sdk.NewCoin(app.BondDenom, sdk.NewInt(1000000)))
+				// HACKHACK this test implicitly depends on other test cases
+				// being run before it because it attempts to spend tokens from
+				// the community pool. Tokens get minted to the community pool
+				// due to inflation so this test may fail if executed in
+				// isolation or the number of tokens that are included in the
+				// spend proposal exceeds the number of tokens minted to the
+				// community pool. To avoid depending on the number of coins
+				// present in the community pool, the spend proposal attempts to
+				// spend 0 coins.
+				coins := sdk.NewCoins(sdk.NewCoin(app.BondDenom, sdk.NewInt(0)))
```

Comment:
@evan-forbes would appreciate your opinion on this

Reply:
:eyes: woah weird! 

I believe this analysis is partly correct, but I think its actually due to how the mint module is calculating how many funds get minted.

because we round down and the blocks are fast, the mint module will never actually mint anything, no matter how long we wait. One would expect this test to pass if we wait for long enough and only send a single utia, but that's not the case. However, if we set the set the block time to 1s and the spend amount to 1utia then the test passes!

I suggest we either:
 - move this test to the mint module integration test
 - set the block time to 1s with a comment + lower spend amount to 1utia
 - keep fast block time and set amount to 0 like above, but also test in the mint module integration test
 - change the way we are calculating inflation per block to not round down nearly as much (see comment below, also this is my preferred option)

Reply:
> change the way we are calculating inflation per block to not round down nearly as much (see comment below, also this is my preferred option)

After doing this, the test passes for a proposal to spend 1utia 🎊